<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="ListaSImple">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>311</width>
          <height>522</height>
         </rect>
        </property>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>281</width>
           <height>211</height>
          </rect>
         </property>
         <property name="title">
          <string>Insertar</string>
         </property>
         <widget class="QLabel" name="PosicionLS_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Posicion</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="editPosLS_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>0, 1, 2, 3...</string>
          </property>
         </widget>
         <widget class="QLabel" name="NumeroLS_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Numero</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="editNumLS_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>0, 1, 2, 3...</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnInsertarLS_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>83</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Insertar</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="gbBorrarLS_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>210</y>
           <width>281</width>
           <height>151</height>
          </rect>
         </property>
         <property name="title">
          <string>Eliminar</string>
         </property>
         <widget class="QLabel" name="PosicionLS_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Posicion</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="editDelPosLS_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>0, 1, 2, 3...</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnBorrarLS_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>83</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Eliminar</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="gbBuscarLS_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>360</y>
           <width>281</width>
           <height>121</height>
          </rect>
         </property>
         <property name="title">
          <string>Busqueda</string>
         </property>
         <widget class="QLabel" name="Buscar_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Numero</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="editFindValLS_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>Value to Find</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnBuscarLS_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>83</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="contenedorGraphicsView" native="true">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>0</y>
          <width>501</width>
          <height>522</height>
         </rect>
        </property>
        <widget class="QGraphicsView" name="graphicsViewLS">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>481</width>
           <height>491</height>
          </rect>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperaciones">
    <property name="title">
     <string>Operaciones</string>
    </property>
   </widget>
   <addaction name="menuOperaciones"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
